### ISSUE TITLE
How to cite

### ISSUE THREAD

**Issue by BeastyBlacksmith** [JuliaNLSolvers/NLsolve.jl#213]
What is the recommended way to cite NLsolve?

**Comment by pkofod** [comment]
Thank you for asking. We don't have anything like a JOSS entry/paper, so maybe I should just set up zenodo for the repo?

**Comment by BeastyBlacksmith** [comment]
I think that would be good and having the `citations.bib` in the repo a la https://github.com/JuliaLang/julia/pull/31794/ is also a good idea

**Comment by pkofod** [comment]
> I think that would be good and having the `citations.bib` in the repo a la [JuliaLang/julia#31794](https://github.com/JuliaLang/julia/pull/31794) is also a good idea

Yes, I saw the discussion about the citations-files going on recently. Is this for a research paper, or?

**Comment by BeastyBlacksmith** [comment]
Yes
